Source/Krypton Components/Krypton.Toolkit/General/KryptonMessageBoxData.cs

```diff
@@ -30,75 +30,84 @@ public struct KryptonMessageBoxData
 
         /// <summary>Gets or sets the buttons.</summary>
         /// <value>The buttons.</value>
-        public KryptonMessageBoxButtons Buttons { get; set; }
+        public KryptonMessageBoxButtons Buttons { get; set; } = KryptonMessageBoxButtons.OK;
 
         /// <summary>Gets or sets the icon.</summary>
         /// <value>The icon.</value>
-        public KryptonMessageBoxIcon Icon { get; set; }
+        public KryptonMessageBoxIcon Icon { get; set; } = KryptonMessageBoxIcon.None;
 
         /// <summary>Gets or sets the default button.</summary>
         /// <value>The default button.</value>
-        public KryptonMessageBoxDefaultButton? DefaultButton { get; set; }
+        public KryptonMessageBoxDefaultButton? DefaultButton { get; set; } = KryptonMessageBoxDefaultButton.Button1;
 
         /// <summary>Gets or sets the <see cref="MessageBoxOptions"/>.</summary>
         /// <value>The <see cref="MessageBoxOptions"/>.</value>
-        public MessageBoxOptions Options { get; set; }
+        public MessageBoxOptions Options { get; set; } = MessageBoxOptions.DefaultDesktopOnly;
 
         /// <summary>Gets or sets the help information.</summary>
         /// <value>The help information.</value>
-        public HelpInfo? HelpInfo { get; set; }
+        public HelpInfo? HelpInfo { get; set; } = null;
 
         /// <summary>Gets or sets the show control copy.</summary>
         /// <value>The show control copy.</value>
-        public bool? ShowCtrlCopy { get; set; }
+        public bool? ShowCtrlCopy { get; set; } = false;
```

Comment:
Why be a `?` if you are initing to a solid value ?

Reply:
Removed default values